scripts/check_compliance.py

```diff
@@ -588,6 +588,122 @@ def run(self):
             self.add_failure(msg.decode("utf-8"))
 
 
+class PyLint(ComplianceTest):
+    """
+    Runs pylint on all .py files, with a limited set of checks enabled
+    """
+    _name = "pylint"
```

Comment:
The other checks are in CamelCase. I don't like CamelCase in CLIs (-m pylint) but I like losing time because of the inconsistency even less.

Reply:
There's `_name = "checkpatch"` too. Could always change them all to lowercase instead.

Reply:
Fears of breaking backward-compatibility and/or CI aside, I'd love everything to become lowercase. Not in this commit anyway.

Reply:
Could I keep it lowercase and lowercase the other ones later? Doesn't seem to be consistent at the moment at least.